pkg/micropython/doc.txt

```diff
@@ -0,0 +1,93 @@
+/**
+ * @defgroup pkg_micropython MicroPython RIOT port
+ * @ingroup  pkg
+ * @brief    MicroPython - Python for microcontrollers
+ *
+ * # MicroPython RIOT package
+ *
+ * "MicroPython is a lean and efficient implementation of the Python 3
+ *  programming language that includes a small subset of the Python standard
+ *  library and is optimised to run on microcontrollers and in constrained
+ *  environments."
+ *
+ * @see https://micropython.org
+ *
+ * ## Status
+ *
+ * MicroPython on RIOT has to be considered experimental. While the basic
+ * interpreter works fairly well on native and Cortex-M, it has not seen much
+ * testing.
+ *
+ * ## Configuration options
```

Comment:
This section should be removed now: it's duplicated with the new one added below.

Reply:
yup. I removed the old one and moved the updated one up.